salt/cloud/clouds/clc.py

```diff
@@ -0,0 +1,368 @@
+# -*- coding: utf-8 -*-
+'''
+CenturyLink Cloud Module
+===================
+
+.. versionadded:: 2017.7.14
+
+The CLC cloud module allows you to manage CLC Via the CLC SDK.
+
+:codeauthor: Stephan Looney <slooney@stephanlooney.com>
+
+
+Dependencies
+============
+
+- clc-sdk Python Module
+
+CLC SDK
+-------
+
+clc-sdk can be installed via pip:
+
+.. code-block:: bash
+
+    pip install clc-sdk
+
+.. note::
+  For sdk reference see: https://github.com/CenturyLinkCloud/clc-python-sdk
+
+Configuration
+=============
+
+To use this module, set up the clc-sdk, username,password,key in the
+cloud configuration at
+``/etc/salt/cloud.providers`` or ``/etc/salt/cloud.providers.d/clc.conf``:
+
+.. code-block:: yaml
+
+    my-clc-config:
+      driver: clc
+      user: 'web-user'
```

Comment:
Are quotes necessary here?

Reply:
usernames/tokens can have special characters that can cause issues with python. I ran into a couple instances in our environment

Reply:
Ah, K. Thanks for the clarification.